packages/node/src/methods/app-instance/uninstall-virtual/controller.ts

```diff
@@ -50,16 +50,14 @@ export default class UninstallVirtualController extends NodeController {
   }
 
   protected async beforeExecution(
+    // @ts-ignore
```

Comment:
The `RequestHandler` should be optional on the `beforeExecution` fn signature to avoid having to ignore it like this.

Reply:
Doesn't actually help because `beforeExecution` gets called by the super class implementation of `enqueueByShard`, but I agree something might be better

Reply:
https://github.com/counterfactual/monorepo/issues/2109